modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceOrders.path

```diff
@@ -215,6 +215,31 @@
 	<td>//a[contains(@href,'${currentOrderId}') and contains(.,'View')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>PRINT_ORDER_TEMPLATE</td>
+	<td>//span[@class='lfr-btn-label' and contains(.,'Select File')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>PRINT_ORDER_TEMPLATE_SELECT_FILE</td>
+	<td>//label[@class='btn btn-secondary' and contains(.,'Select File')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>PRINT_ORDER_TEMPLATE_FILE_NAME</td>
+	<td>//span[@id='_com_liferay_commerce_channel_web_internal_portlet_CommerceChannelsPortlet_fileEntryNameInput' and contains(.,'${value1}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>PLACED_ORDERS_ELLIPSIS</td>
+	<td>//a[@class='component-action direction-right dropdown-toggle btn btn-lg btn-monospaced btn-primary position-fixed thumb-menu']</td>
```

Comment:
This looks prone to failure due to the long @class.
check [icon.path](https://github.com/liferay/liferay-portal/blob/master/portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Icon.path) for something reusable.